envs/m85-an555/Makefile

```diff
@@ -36,6 +36,8 @@ CFLAGS += \
 	$(ARCH_FLAGS) \
 	--specs=nosys.specs
 
+CFLAGS += ${CFLAGS_EXTRA}
```

Comment:
Why is this {} and not () ?

Reply:
No particular reason -- If you like `(...)` better, I can change it.